permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java

```diff
@@ -125,6 +125,10 @@ public boolean onRequestPermissionsResult(int requestCode, String[] permissions,
            return false;
         }
 
+        if (permissions.length ==0 && grantResults.length==0){
```

Comment:
Could you add a `Log.w` call here that tells the developer that this callback was called without results?

Your changes make sure the application will not crash. Also, this case should never happen. But in case it does, like in the issue you reported in the comments, we would like the developer to be aware of it so they can raise an issue. Otherwise we risk that the plugin fails silently, which can be quite confusing when debugging.